dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditStatus.java

```diff
@@ -75,56 +77,71 @@ public PublishAuditStatus(PublishAuditStatus origin, Status status) {
 		this.createDate = origin.createDate;
 		this.statusPojo = origin.getStatusPojo(); //TODO manage status POJO
 	}
-	
+
+	/**
+	 * Return true if  any of the endpoint has the {@link Status#INVALID_TOKEN}
+	 * @return
+	 */
+	public boolean isAnyEndPointInvalidTokenStatus() {
```

Comment:
rename to `doesAnyEndpointHaveInvalidTokenStatus`

Reply:
really this method is not used

https://github.com/dotCMS/core/pull/19273/commits/303cd0bf044e9265865287ea596719a4346e2895#diff-470260e643cc175a1a0c76a8e25eb171L85